tests/sites/doxygen/extension/conf.py

```diff
@@ -0,0 +1,10 @@
+html_theme = "rocm_docs_theme"
+extensions = ["rocm_docs", "rocm_docs.doxygen"]
+
+doxygen_project = {
+    "name": "Rocm Docs Core Doxygen Legacy Test Project",
+    "path": ".doxygen/docBin/xml",
```

Comment:
No they are somewhat orthogonal. `doxygen_project` is used to set up [breathe](https://github.com/breathe-doc/breathe), it will be translated to [`breathe_projects`](https://breathe.readthedocs.io/en/latest/directives.html#confval-breathe_projects) basically. This is mainly an interface that the legacy (`import ROCMDocs`) interface can use for backwards compatibility. I would like to remove this option in the future, projects could just set up breathe themselves, there really isn't anything of value by doing it in rocm-docs-core. That's why I decided to make it a different variable, instead of adding a new key to doxygen_project.

`doxygen_html` is intended to be used when a project is using doxysphinx. The `path` in `doxygen_project` needs to point to the xml output of doxygen, `doxygen_html` to the HTML output.